docs/docs/deployment/hosting/locally-edge-proxy.md

```diff
@@ -229,7 +235,39 @@ Or, log access logs to file in generic format while logging everything else to s
 When adding logger configurations, you can use the `"default"` handler which writes to stdout and uses formatter
 specified by the [`"logging.log_format"`](#logginglog_format) setting.
 
-### `config.json` example
+### Health Check
+
+The health check can be configured depending on the use case for the Edge Proxy by adding the `health_check` object to
```

Comment:
Suggest using active voice instead of passive voice https://developers.google.com/style/voice, and clarifying what the default behaviour is. Something like this:

```
The Edge Proxy exposes a health check endpoint at `/proxy/health` that responds with a 200 status code if was able to fetch all its configured environment documents. If any environment document could not be fetched during the latest poll, it will respond with a 500 status code.

In some cases, you may want the Edge Proxy to succeed its health checks even if it failed to fetch any environment document, but only if it these documents were successfully fetched at some point in the past. You can achieve this by...
```

(I was typing this suggestion until I saw this: https://github.com/Flagsmith/flagsmith/pull/4580/files#r1743739010 - this would need to be changed depending on what the new defaults are)

Reply:
Good feedback, thanks. I've updated the PR with changes that (I think) resolve both of your comments.